content/backmatter.tex

```diff
@@ -829,72 +856,72 @@ \section{Version 1.4}
 \ChangelogRef{dep:overview, dep:rationale}%
 %
 \item Deprecated header directory \HEADER{mpp}.
-\\ See Section~\ref{dep:mpp_header}.
+\ChangelogRef{dep:mpp_header}%
 %
 \item Deprecated the \FUNC{shmem\_wait} functions and the \CTYPE{long}-typed \CorCpp \FUNC{shmem\_wait\_until} function.
-\\ See Section \ref{subsec:p2p_intro}.
+\ChangelogRef{dep:shmem_wait, dep:long_typed_shmem_wait_until}%
```

Comment:
This is another case where we have lost a reference.

Reply:
I'll change this pointer to include `subsec:shmem_wait_until` instead of `subsec:p2p_intro`.

Reply:
:+1: